produce.go

```diff
@@ -97,6 +97,17 @@ func (cmd *produceCmd) run(args []string) error {
 	cmd.partitioner = partitioner
 	var err error
 
+	if cmd.avroSchemaID != 0 {
```

Comment:
does the unique ID start at 1 ?

Reply:
it may start with 0 according to https://github.com/confluentinc/schema-registry/blob/76505e717760a06bbd7956e1bde242a6d265bb2f/core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java#L504 , I can change to use a pointer to make sure 0 works too but in practical terms it starts with 1